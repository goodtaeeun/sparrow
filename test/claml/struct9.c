typedef struct bfd bfd;

struct bfd {
  int cacheable;
};

void f(bfd *b) { b->cacheable = 1; }
