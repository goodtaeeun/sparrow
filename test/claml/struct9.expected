/* Generated by CIL v. 1.7.3 */
/* print_CIL_Input is false */

#line 1 "struct9.c"
struct bfd;
#line 1 "struct9.c"
typedef struct bfd bfd;
#line 3 "struct9.c"
struct bfd {
  int cacheable;
};
#line 7 "struct9.c"
void f(bfd *b) {

  {
#line 7
    b->cacheable = 1;
    return;
  }
}
